contractdata/HOKKAIDO/HOKKAIDOSNOWFESTIVAL.json

```
@@ -27,11 +44,11 @@
         "Description": "UI_CONTRACT_MAMUSHI_TARGET_BRIEFING",
         "BriefingVideo": "briefing_mamushi",
         "DebriefingVideo": "debriefing_mamushi",
-        "Location": "LOCATION_HOKKAIDO_SHIM_MAMUSHI",
+        "Location": "LOCATION_HOKKAIDO",
```

```suggestion
        "Location": "LOCATION_HOKKAIDO_SHIM_MAMUSHI",
```
Peacock-specific behavior